### DISCUSSION TITLE
I can't load new version of Osu!Lazer

### DISCUSSION THREAD

**Discussion by Sewlell** [ppy/osu#15534] [Q&A] [answered]
Welp... I tried to play 2021.1108.0 version of osu!lazer but it doesn't load. I see my osu!lazer working in 10 seconds and it is gone from my Task Manager. I tried so many time that it doesn't working so I switch to older version ( 2021.523.0 ) and it work ( then you know, auto-update and not working again due to new version ).

During that time, I also seeing some former issue that is similar from my issue. But, One, I already download KB3063858. Two, my .NET SDK was the newest version which is 6.0.100. Three, I am using 64 bit Windows 7 ( Service Pack 1 ). Four, Event viewer? well....

Here my specs :
Processor : Intel Core i3-2370M @ 2.40GHz
RAM : 2GB
I have .NET Framework 4.8
Graphic Card : Intel (R) HD Graphic 3000

Keep in mind that my mother doesn't wanted me to update my computer's Windows Defender and also Windows Update because my Windows is rooted by some manufactuer. So the only thing I could do is just update my processor drivers, .NET Core and Framework.

And now, right here, is the thing from %appdata%/osu/logs
[database.log](https://github.com/ppy/osu/files/7502860/database.log)
[network.log](https://github.com/ppy/osu/files/7502862/network.log)
[performance.log](https://github.com/ppy/osu/files/7502863/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7502864/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7502865/updater.log)

If you wanted more information, please tell me.

**Comment by bdach** [comment, marked as answer]
Support for windows 7 was dropped. I can only recommend updating to a newer OS.

**Reply by Sewlell** [reply]
Alright, thank for answering my issue.

**Comment by Carprai** [comment]
Just came here to post about an extremely similar problem, I'm trying to set up some stuff for the lazer tournament client but the new version of lazer seems to immediately open and then close. I can't open an issue either unfortunately so I wasn't able to report it. I don't know if it's the exact same problem but I figured I'd ask about it.

I only have runtime and update_success logs, idk if they'd be any help but let me know (I'm new to reporting bugs so idk what you guys need specifically.)

Here's what the Event Viewer says: https://pastebin.com/Rz5Uy1Ru

(Forgot to mention I'm on Windows 10. Running an i5-7600k and a GTX 1060)

**Reply by bdach** [reply]
your problem is actually entirely different. please see https://github.com/ppy/osu/discussions/15530 for instructions on how to temporarily bypass. a fix is in the works.

**Reply by Carprai** [reply]
Ooh okay, didn't see that, thank you!
